content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -159,6 +160,66 @@ does not actually persist the TokenReview object into etcd.
 Hence `kubectl get tokenreview` is not a valid command.
 {{< /note >}}
 
+#### Schema for service account private claims
+
+The schema for the Kubernetes-specific claims within JWT tokens is not currently documented,
+however the relevant code area can be found in [the serviceaccount package]() in the Kubernetes codebase.
+
+[the serviceaccount package]: https://github.com/kubernetes/kubernetes/blob/d8919343526597e0788a1efe133c70d9a0c07f69/pkg/serviceaccount/claims.go#L56-L68
+
+You can inspect a JWT using standard JWT decoding tools, an example of a JWT for the
+`my-serviceaccount` ServiceAccount, bound to a Pod object named `my-pod` which is scheduled
+to the Node `my-node`, in the `default` namespace:
+
+```json
+{
+  "aud": [
+    "https://kubernetes.default.svc.cluster.local"
```

Comment:
make it clear with a comment that this is an example and can vary from cluster to cluster, and from token to token (you can switch the formatting type to yaml and add # comments that will get formatted nicely) (or better yet, make this clearly a dummy value like `https://my-audience.example.com`)